docs/UserGuide.md

```diff
@@ -259,10 +281,6 @@ our definition of identical internships further, Flagship will not be able to ca
 as effectively.
 </div>
 
-<div markdown="block" class="alert alert-info">
-ℹ️ We allow you to create internship entries with application deadlines and start dates before today's current date. It is designed as such 
-so that you can track your past internship applications for your own reference. 
-</div>
 
```

Comment:
Is there a reason you deleted this? I inserted this specifically to guard against PE attacks

Reply:
added back